docs/features/anomaly-detection/metrics.mdx

```diff
@@ -84,10 +84,10 @@ models:
             type: row_count
           - name: filtered_row_count
             type: row_count
-            where_expression: country = 'USA'
           - name: null_count
             type: null_count
             columns: [hello, world]
+        where_expression: country = 'USA'
         cloud_monitored=True
```

Comment:
Also this seems invalid YAML

Reply:
<img src="https://media3.giphy.com/media/OJp5pne69OeY/giphy.gif"/>